server/start.go

```diff
@@ -448,9 +448,11 @@ func startInProcess(svrCtx *server.Context, clientCtx client.Context, opts Start
 		indexerService := NewEVMIndexerService(idxer, clientCtx.Client.(rpcclient.Client))
 		indexerService.SetLogger(servercmtlog.CometLoggerWrapper{Logger: idxLogger})
 
-		g.Go(func() error {
-			return indexerService.Start()
-		})
+		go func() {
+			if err := indexerService.Start(); err != nil {
```

Comment:
just curious, should we make indexerService to honor the context.Context provided by the errgroup and gracefully shutdown when the context is canceled?

Reply:
but other api service is already shutting down, seems no need to wait